e2e/e2e.go

```diff
@@ -74,21 +74,41 @@ func BuildDependenciesList(ctx context.Context) ([]string, error) {
 	return depsList, nil
 }
 
+// getConfig gets config file from GitHub and parses it into DevenvConfig
+func getConfig(ctx context.Context, serviceName string, gh *github.Client, configFileName string) (*DevenvConfig, error) {
+	r, _, err := gh.Repositories.DownloadContents(ctx, "getoutreach", serviceName, configFileName, nil)
```

Comment:
Change `getoutreach` to be the org from the box config?

Reply:
Fixed, yeah, I was too lazy